simple-backend/stave_backend/lib/stave_cli.py

```diff
@@ -68,57 +85,58 @@ def get_args():
-                                help="Path to log file for logging")
+    for opt in CONFIG_ARGS.values():
+        parser_config.add_argument(*opt["name_or_flags"], help=opt["help"])
 
     return parser.parse_args()
 
-def set_logger(verbose: bool, log_file: str):
+def set_logger_verbose(verbose: bool):
     """
-    Set up logging. The implementation is contingent on the LOGGING field
-    in "stave_backend/settings.py", so this can only be called after
+    Set up logging verbose at runtime. The implementation is contingent on
+    static LOGGING setting in "stave_config.py". This can only be called after
     the server starts.
     """
     root_logger = logging.getLogger()
-    root_logger.setLevel(logging.NOTSET)
-
-    stream_handler = root_logger.handlers[0]
-    stream_handler.setLevel(logging.INFO if verbose else logging.ERROR)
-
-    file_handler = logging.FileHandler(log_file)
-    file_handler.setLevel(logging.NOTSET)
-    file_handler.setFormatter(stream_handler.formatter)
 
-    root_logger.addHandler(file_handler)
-    logging.getLogger("django").addHandler(file_handler)
+    root_handlers = root_logger.handlers
+    if root_handlers and isinstance(root_handlers[0], logging.StreamHandler):
+        root_handlers[0].setLevel(logging.INFO if verbose else logging.ERROR)
+    else:
+        root_logger.setLevel(logging.INFO if verbose else logging.ERROR)
 
     return root_logger
 
 def main():
     
     args = get_args()
+    config = StaveConfig()
     in_viewer_mode = args.command == START and args.project_path is not None
     thread_daemon = not (args.command == START)
 
     if args.command == CONFIG:
-        change_config = (args.db_file or args.log_file) is not None
-        if change_config:
-            StaveViewer.set_config(
-                db_file=args.db_file and os.path.abspath(args.db_file),
-                log_file=args.log_file and os.path.abspath(args.log_file)
-            )
-        if args.show_config or (not change_config):
-            print(json.dumps(
-                StaveViewer.load_config(),
-                sort_keys=True,
-                indent=4
-            ))
+        for field in CONFIG_ARGS.keys():
+            if getattr(args, field) is not None:
+                setattr(config, field, getattr(args, field))
+        config.show_config()
         sys.exit()
 
+    # Interactively set up the configuration
+    if not config.is_initialized():
+        print(
+            "\nInitialize Stave configuraion\nEnter the entry for each "
+            "prompt below.\nYou may leave empty to accept the default value."
+            "\nNote: If you set the 'django_settings_module' field, the "
+            "other configs (e.g., 'db_file', 'log_file') will be ignored."
+        )
+        for name, opt in CONFIG_ARGS.items():
+            val: str = input(
+                f"\n<name: {name}>\n<description: {opt['help']}>\n"
+                f"<default: {getattr(config, name)}>\n> "
```

Comment:
The default value for the config ` django_settings_module` is None, maybe it is a bit confusing?